x-pack/plugins/observability/public/pages/rule_details/index.tsx

```diff
@@ -536,22 +532,20 @@ export function RuleDetailsPage() {
           onSave: reloadRule,
         })}
       <DeleteModalConfirmation
-        onDeleted={async () => {
+        onDeleted={() => {
```

Comment:
@fkanout I checked the implementation at `triggers_action_ui` plugin and it uses `async` there https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_details.tsx#L269. In the future we should use a shareable delete modal confirmation, so this implementation will be removed anyway. I was just wondering what happens if you use their implementation. Does it still work?

Reply:
@mgiota, we were using that implementation. Nothing changes as there is no `await`.